.github/test-scenarios.yml

```diff
@@ -0,0 +1,45 @@
+# Integration test scenario definitions for building test matrix
+
+# Scenario configuration format:
+# - name: Required string identifier for the scenario
+# - description: Optional string description (defaults to name if not provided)
+# - tox_env: Optional string specifying the tox environment to run
+# - needs_trust: Optional boolean for workload identity trust setup (default: false)
+# - init_args: Optional string with extra arguments for 'ops init' command
+# - create_args: Optional string with extra arguments for 'ops create' command
+# - test_redeploy: Optional boolean to test redeployment in the scenario (default: false)
+# - env: Optional list of environment variable dictionaries with key-value pairs
+# - parallel: Optional boolean controlling parallel execution in pytest-xdist (default: true)
+
+scenarios:
+  - name: edge
+    description: "Edge tests"
+    tox_env: python-edge-int
+
+  - name: insecure-listener
+    description: "Insecure listener deploy, long running tests"
```

Comment:
Looking at your run posted in the PR long running tests are not much longer than cloud tests or wlif tests 😄